### PR TITLE
Add labs section

### LINKED ISSUE #1: Add labs section
The main page should contain labs sections for students to understand what's ahead.

### PR DESCRIPTION
**Pull request by voievodin**
Resolves #1. 

Since the view is bad on smalls screens the pr also makes the view full-with. Best to review commit by commit.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,8 +27,6 @@
     
     <div class="container-fluid">
       <div class="col-md-8">
-    
-      <div class="col-md-5">
```
Comment (yevhenii0) [inline index.html:31]: I think it's better to make it inlined block, so that the width is content wide.

File: index.html [hunk 2]
```diff
@@ -55,11 +53,15 @@ <h1>Lectures</h1>
...
     
-      </div>
-    
-      <div class="col-md-7">
-      </div>
+      <ul class="list-group list-group-numbered">
+        <li class="list-group-item">Git</li>
+        <li class="list-group-item">Build systems</li>
+        <li class="list-group-item">Design patterns</li>
+        <li class="list-group-item disabled">IoC</li>
```
Comment (yevhenii0) [inline index.html:63]: It shouldn't have been disabled!

File: index.html (hunk 1 above)
Comment (voievodin) [reply index.html:31]: Ah good idea :+1:

File: index.html (hunk 2 above)
Comment (voievodin) [reply index.html:63]: It must have left after me testing things. Thanks for noticing!

**Review by yevhenii0 — APPROVED** [review]

**Pull request merged**